cf-deployment.yml

```diff
@@ -305,6 +305,7 @@ instance_groups:
             override: true
             refresh-token-validity: 2592000
             scope: cloud_controller.read,cloud_controller.write,openid,password.write,cloud_controller.admin,scim.read,scim.write,doppler.firehose,uaa.user,routing.router_groups.read,routing.router_groups.write
+            secret: ''
```

Comment:
@dsabeti the `cf` client uses a password grant. This means the client has to have a password.
In most situations, clients that cannot protect its credentials use the `implicit` grant. But this is a browser flow, thus we have the situation where the `cf` client is a command line application, but cannot protect its credentials.

The compromise was to have the client use the `password` flow and hard code the password to an empty string `""`. 

If you change the secret of the `cf` client then I believe `cf cli` application will stop working, unless it has the option to set the client password.